game/src/info/person.rs

```diff
@@ -154,7 +154,7 @@ pub fn trips(
                     .padding_bottom(2),
             ])
             .centered()
-            .fully_rounded()
+            .corner_rounding(CornerRounding::FullyRounded)
```

Comment:
While moving the CornerRounding from a single float to potentially 4 floats, I made the FullyRounded rather than a `Option::None<f64>`.

Reply:
Nice! `None` was always a hack.